src/api/rateArticleApi.js

```diff
@@ -0,0 +1,36 @@
+import instance from "./axiosConfig";
+
+export default class rateArticleApi {
+  static rateArticle(rated, slug) {
+    return instance
+      .post(
+        `/api/article/${slug}/rate`,
+        {
+          article: {
+            rate: rated
+          }
+        }
+      )
+      .then((response) => {
+        if (response.status === 201) {
+          return { success: true, articleRating: response.data };
+        }
+      })
+      .catch((err) => {
+        throw err;
+      });
+  }
+
+  static getArticleRate(slug) {
+    return instance
+      .get(`/api/article/${slug}/rating`)
+      .then((response) => {
+        if (response.status === 200) {
+          return { success: true, articleRating: response.data["Rated at"] };
+        }
+      })
+      .catch((err) => {
+        throw err;
```

Comment:
Please handle the catch clause well. Add a toastr message for example.

Reply:
Cool. on it